server/apps/postgres-app/README.adoc

```diff
@@ -0,0 +1,144 @@
+= Guice-JPA Server How-to
```

Comment:
Guice-Postgres Server How-to

And a TODO (modify the readme later upon having proper Postgres app?)